src/sentry/api/endpoints/organization_projects.py

```diff
@@ -102,17 +102,26 @@ def get(self, request, organization):
 
         queryset = queryset.filter(status=ProjectStatus.VISIBLE).distinct()
 
-        return self.paginate(
-            request=request,
-            queryset=queryset,
-            order_by=order_by,
-            on_results=lambda x: serialize(
-                x,
-                request.user,
-                ProjectSummarySerializer(
-                    environment_id=self._get_environment_id_from_request(request, organization.id),
-                    stats_period=stats_period,
+        # TODO(davidenwang): remove this after frontend requires only paginated projects
+        get_all_projects = request.GET.get("all_projects") == "1"
+
+        if get_all_projects:
+            sorted_projects = sorted(list(queryset), key=lambda x: x.slug)
```

Comment:
not sure if this projects list definitely needs to be sorted, but the intention is to copy what `DetailedOrganizationSerializerWithProjectsAndTeams` is doing

Reply:
Probably better to sort it in the query rather than in python